include/balltze_transport.h

```diff
@@ -2,11 +2,12 @@
 #define BALLTZE_TRANSPORT
 
 #include <Arduino.h>
-#include <stdio.h>
+
 #include <stdbool.h>
+#include <stdio.h>
 #include <sys/time.h>
 
-int clock_gettime(clockid_t unused, struct timespec *tp);
+int clock_get_time(clockid_t unused, struct timespec *tp);
```

Comment:
I have changed that to be more consistent with naming convention

Reply:
Not sure if it is actually used anywhere else in the project. As far as I see its not, maybe we don't need it at all 🦘

Reply:
This is extern function required by POSIX. This has to have this name